src/Periodic/LargePeriodicHTMLFormatter.php

```diff
@@ -0,0 +1,231 @@
+     * @param \CultuurNet\SearchV3\ValueObjects\Offer|\CultuurNet\SearchV3\ValueObjects\Place $place
+     * @return mixed
+     */
+    public function format($place)
+    {
+        $output = $this->generateDates($place->getStartDate(), $place->getEndDate());
+
+        if ($place->getOpeningHours()) {
+            $output .= $this->generateWeekScheme($place->getOpeningHours());
+        }
+
+        return $this->formatSummary($output);
+    }
+
+    /**
+     * @param $calsum
+     * @return mixed
+     */
+    protected function formatSummary($calsum)
+    {
+        $calsum = str_replace('><', '> <', $calsum);
+        return str_replace('  ', ' ', $calsum);
+    }
+
+    /**
+     * @param $time
+     * @return string
+     */
+    protected function getFormattedTime($time)
+    {
+        $formattedShortTime = ltrim($time, '0');
+        if ($formattedShortTime == ':00') {
+            $formattedShortTime = '0:00';
+        }
+        return $formattedShortTime;
+    }
+
+    /**
+     * Retrieve the earliest time for the specified daysOfWeek.
+     *
+     * @param $openingHoursData
+     * @param $daysOfWeek
+     * @return string
+     */
+    protected function getEarliestTime($openingHoursData, $daysOfWeek)
+    {
+        $earliest = '';
+        foreach ($openingHoursData as $openingHours) {
+            if ($daysOfWeek === $openingHours->getDayOfWeek()) {
+                if (!empty($earliest)) {
+                    $earliest = ($openingHours->getOpens() < $earliest ? $openingHours->getOpens() : $earliest);
+                } else {
+                    $earliest = $openingHours->getOpens();
+                }
+            };
+        }
+        return $earliest;
+    }
+
+    /**
+     * Retrieve the latest time for the specified daysOfWeek.
+     *
+     * @param $openingHoursData
+     * @param $daysOfWeek
+     * @return string
+     */
+    protected function getLatestTime($openingHoursData, $daysOfWeek)
+    {
+        $latest = '';
+        foreach ($openingHoursData as $openingHours) {
+            if ($daysOfWeek === $openingHours->getDayOfWeek()) {
+                if (!empty($latest)) {
+                    $latest = ($openingHours->getCloses() > $latest ? $openingHours->getCloses() : $latest);
+                } else {
+                    $latest = $openingHours->getCloses();
+                }
+            };
+        }
+        return $latest;
+    }
+    
+    /**
+     * @param DateTime $dateFrom
+     * @param DateTime $dateTo
+     * @return string
+     */
+    protected function generateDates(DateTime $dateFrom, DateTime $dateTo)
+    {
+        $fmt = new IntlDateFormatter(
+            'nl_BE',
+            IntlDateFormatter::FULL,
+            IntlDateFormatter::FULL,
+            date_default_timezone_get(),
+            IntlDateFormatter::GREGORIAN,
+            'd MMMM yyyy'
+        );
+
+        $dateFromTimestamp = $dateFrom->getTimestamp();
+        $intlDateFrom =$fmt->format($dateFromTimestamp);
```

Comment:
Intl also supports dateTime objects. This works also:
        $intlDateFrom = $dateFormatter->format($dateFrom);
        $intlDateTo = $dateFormatter->format($dateTo);